cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -45,6 +46,26 @@ FIREBOLT_CALL = {
   content: ...
 };
 ```
+### How to initialize the Firebolt object
+- The Firebolt object should be initialized using the [the environment has been set up for {string} tests](../../support/step_definitions/testSetup.md#the-environment-has-been-set-up-for-string-tests) glue code.
+- The glue code will accept the test type as a parameter. The test type should be the key name of the Firebolt object.
+- The Firebolt object will be fetched based on the key name and stored in the `runtime` environment variable.
+
+#### The are two ways to initialize the Firebolt object
+- Passing the key name in `module:method` format
```

Comment:
what about space in between? Accessibility Closedcaptionsettings

Reply:
Added doc for firebolt call key with space and it will works as is